### PR TITLE
Some random fixes

### PR DESCRIPTION
**Pull request by Yhprum**
Some things I noticed since my last changes
- Links now format properly since slack decided to invent their own markdown format
- The token detecting and deleting from public channels has been reworked to work with cookies
- I tried to paste my json blob in on a mac and slack formatted my quote to curly quotes so I made a change to accept them anyway
- `emojme status` now returns `lastRefresh` as the correct time instead of the current time

### CODE REVIEW

**Comment by Yhprum** [comment]
for some reason 1c05567ca626b6df5943de09ab55d2b0510cd8bd isn't being picked up 😞

File: src/util.coffee [hunk 1]
```diff
@@ -157,7 +172,7 @@ module.exports = (robot) ->
     self = this
     self.readAdminList (emojiList) ->
       if (
-        (emojiList = robot.brain.get 'emojme.AdminList' ) &&
```
Comment (jackellenberger) [inline src/util.coffee:160]: does cacheing in the robo brain not work anymore?

File: src/util.coffee [hunk 2]
```diff
@@ -13,10 +14,24 @@ one_day = 1000 * 60 * 60 * 24
...
     if request.message.room[0] == 'C'
-      slack.chat.delete({token: token, channel: request.message.room, ts: request.message.id})
-      request.send("Don't go posting slack auth tokens in public channels ya dummy. Delete that or I'm telling mom.")
+      try
+        authJson = JSON.parse(authJsonString)
+        token = authJson["token"]
+        cookie = authJson["cookie"]
+        subdomain = authJson["domain"] || authJson["subdomain"] || request.message.user.slack.team_id.replace(/:/g,'').trim()
+        slack = new SlackClient(subdomain, cookie)
+        if token && cookie
```
Comment (jackellenberger) [inline src/util.coffee]: I think the pedantic swe in me wants to say there's a way to invert this conditional tree to `fail fast`, e.g. 
```
if !(token && cookie)
    fail message
parse out (response) ->
    delete and chastise
    return
tell mom
```

that said i _do not care_ and _hate software_

**Review by jackellenberger — APPROVED** [review]
This is excellent, thank you!

Again I gotta wait until i'm off of company networking to get npm to work, but i'll go forward with it tonight if umm i remember. 

One real question to address re the brain - i don't remember what state i left that in tbh, it's possible it doesn't work at all idk

File: src/util.coffee (hunk 1 above)
Comment (Yhprum) [reply src/util.coffee:160]: tbh I haven't really looked into how caching works here at all, but the emojiList variable being returned by `self.readAdminList` is already trying the cache so it seemed redundant to me (correct me if I'm wrong)

File: src/util.coffee (hunk 2 above)
Comment (Yhprum) [reply src/util.coffee]: It was kinda painful writing the same message for every fail path so I like your idea much more - I can make a change for this

**Comment by Yhprum** [comment]
Okay I added 1 more bugfix I just found. TIL that `Date()` refuses to take any arguments and you have to use `new Date(lastRefresh)`. Looks like this has been broken since https://github.com/jackellenberger/emojme-hubot-plugin/commit/566709ed77e1ddc70996049e9e9b05f639428c24 but who ever uses the status command except right after refreshing anyway?

File: src/commands.coffee [hunk 3]
```diff
@@ -75,7 +74,7 @@ Questions, comments, concerns? Ask em either on emojme, or on [this project](htt
     authJsonString = authResponse.match[1].trim()
     try
       authJson = JSON.parse(authJsonString)
-      token = authJson["domain"] || authJson["subdomain"]
```
Comment (jackellenberger) [inline src/commands.coffee:78]: literally embarrassing

**Comment by jackellenberger** [comment]
this is finally released as https://www.npmjs.com/package/emojme-hubot-plugin/v/1.0.1 sorry so much for the delay!!

**Pull request merged**